x/gamm/keeper/migrate_test.go

```diff
@@ -98,25 +93,12 @@ func (suite *KeeperTestSuite) TestMigrate() {
 				sender:                defaultAccount,
 				sharesToMigrateDenom:  defaultGammShares.Denom,
 				sharesToMigrateAmount: invalidGammShares.Amount,
-				poolIdEntering:        2,
 			},
 			sharesToCreate:         defaultGammShares.Amount,
 			expectedPosition:       &model.Position{Liquidity: sdk.MustNewDecFromStr("100000000000.000000010000000000")},
 			setupPoolMigrationLink: true,
 			expectedErr:            sdkerrors.Wrap(sdkerrors.ErrInsufficientFunds, fmt.Sprintf("%s is smaller than %s", defaultGammShares, invalidGammShares)),
 		},
-		{
-			name: "error: poolIdEntering is not the canonical link",
-			param: param{
-				sender:                defaultAccount,
-				sharesToMigrateDenom:  defaultGammShares.Denom,
-				sharesToMigrateAmount: defaultGammShares.Amount,
-				poolIdEntering:        1,
-			},
-			sharesToCreate:         defaultGammShares.Amount,
-			setupPoolMigrationLink: true,
-			expectedErr:            types.InvalidPoolMigrationLinkError{PoolIdEntering: 1, CanonicalId: 2},
-		},
```

Comment:
Note to reviewer: This test case is no longer possible due to user no longer being the one to provide link information